styles.css

```diff
@@ -46,6 +51,27 @@ body {
 .color-format {
   height: 180px;
   width: 180px;
+  border: black solid 5px;
+}
+
+.color-1 {
+  background-color: #FFFC31;
```

Comment:
Good idea to hardcode these colors to check that they filled the boxes correctly!